.github/workflows/on-pull-request-run-tests.yml

```diff
@@ -127,7 +127,7 @@ jobs:
           install: false
           record: true
           parallel: true
-          auto-cancel-after-failures: 1
+          # auto-cancel-after-failures: 1
```

Comment:
The docs mention this variable can be set to "false" to disable auto-cancellation for the CLI run: https://docs.cypress.io/guides/cloud/smart-orchestration/run-cancellation#Specifying-failures-before-auto-canceling-in-the-CLI